devel/gk/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    gk
+version                 1.0.6
+
+categories              devel
+platforms               darwin
+license                 CC-BY-3.0-US
+installs_libs           no
+maintainers             sergiolms \
+                        alejandroSuch \
+                        JBUinfo
+
+PortGroup               github 1.0
```

Comment:
As mentioned previously, this should move near the top of the file near the `PortSystem` line.